enclave/src/lib.rs

```diff
@@ -313,12 +318,12 @@ pub unsafe extern "C" fn get_state(
     if !requests.is_empty() {
         let responses: Vec<WorkerResponse<Vec<u8>>> = match worker_request(requests) {
             Ok(resp) => resp,
-            Err(e) => return e,
+            Err(e) => return e.into(),
         };
 
         let update_map = match verify_worker_responses(responses, latest_header) {
             Ok(map) => map,
-            Err(e) => return e,
+            Err(e) => return e.into(),
         };
```

Comment:
Is there a sepcific reason not to use `map_err()` here?

Reply:
This is not obvious in the diff, but this function does return a plain `sgx_status_t`, not a `Result`.

Reply:
Ah right, could've seen that because the statement has no `?;` 👍